devtools/x/src/test.rs

```diff
@@ -76,7 +79,10 @@ pub fn run(mut args: Args, xctx: XContext) -> Result<()> {
     if args.lib {
         direct_args.push(OsString::from("--lib"));
     }
-
+    if args.jobs.is_some() {
```

Comment:
Use and “if let” here instead of unwrapping below

Reply:
@bmwill  Not sure what you're asking here,  

if let jobs = if args.jobs.??? {
}

Reply:
I'm just going to land for now, so folks aren't blocked.   But I would like to figure out what I should have done here instead.

Reply:
pattern matching is a thing, as is Some(var) = Option(somevar) {....  okay interesting.